src/App.tsx

```diff
@@ -34,8 +39,15 @@ const App: React.FC = () => {
               />
               <Route path='/profile' element={<MemberProfilePage />} />
               <Route path='/join-swecc' element={<JoinPage />} />
+              <Route path='/login' element={<LoginPage />} />
+              <Route path='/logout' element={<LogoutPage />} />
+              <Route path='/register' element={<RegisterPage />} />
               <Route path='/' element={<div>Home</div>} />
               <Route path='*' element={<div>Not Found</div>} />
+              <Route path="/protected" element={
```

Comment:
Could be cool to have a similar HLC for development only components used for debug, like `<DebugRoute>...</DebugRoute>`